### PR TITLE
velero: add page

### PR DESCRIPTION
**Pull request by Aracki**
I have added [velero](https://github.com/heptio/velero) - a tool for managing k8s backups by Heptio.

### CODE REVIEW

File: pages/common/velero.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# velero
+
+> Backup and migrate Kubernetes applications and their persistent volumes.
+> More informations: <https://github.com/heptio/velero>.
```
Comment (pxgamer) [inline pages/common/velero.md]:
```suggestion
> More information: <https://github.com/heptio/velero>.
```

File: pages/common/velero.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`velero backup get`
+
+- Delete a backup:
+
+`velero backup delete {{backup_name}}`
+
+- Create a weekly backup, each living for 90 days (2160 hours):
+
+`velero schedule create {{schedule_name}} --schedules="@every 7d" --ttl 2160h0m0s`
```
Comment (pxgamer) [inline pages/common/velero.md]:
```suggestion
`velero schedule create {{schedule_name}} --schedules="{{@every 7d}}" --ttl {{2160h0m0s}}`
```

File: pages/common/velero.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Delete a backup:
+
+`velero backup delete {{backup_name}}`
+
+- Create a weekly backup, each living for 90 days (2160 hours):
+
+`velero schedule create {{schedule_name}} --schedules="{{@every 7d}}" --ttl {{2160h0m0s}}`
+
+- Create a restore from the latest successful backup triggered by specific schedule:
```
Comment (andrik) [inline pages/common/velero.md]:
```suggestion
- Restore the latest successful backup triggered by specific schedule:
```

Comment (Aracki) [reply pages/common/velero.md]: It is actually creating a restore "object". All the restores can be listed with : `velero restore get`

Comment (andrik) [reply pages/common/velero.md]: I see, ok then lgtm

**Pull request merged**